misc/python/materialize/mzcompose/services/mysql.py

```diff
@@ -13,20 +13,30 @@
 )
 
 
-class MySql(Service):
-    DEFAULT_ROOT_PASSWORD = "p@ssw0rd"
-    DEFAULT_VERSION = "8.0.35"
-
-    DEFAULT_ADDITIONAL_ARGS = [
+def create_mysql_server_args(server_id: str, is_master: bool) -> list[str]:
+    args = [
         "--log-bin=mysql-bin",
         "--gtid_mode=ON",
         "--enforce_gtid_consistency=ON",
         "--binlog-format=row",
-        "--log-slave-updates",
         "--binlog-row-image=full",
-        "--server-id=1",
+        f"--server-id={server_id}",
     ]
 
+    if is_master:
+        args.append("--log-slave-updates")
```

Comment:
this is only applicable on a replica, not the source-server. It says whether the replica will record the events received by the source into the replica's own binary log

Reply:
I think it's defaulted to ON which is why it likely isn't having an impact on the tests

Reply:
✅ Adjusted